src/stringClass.js

```diff
@@ -1,33 +1,165 @@
 const stringClass = {
+  /**
+   * This method takes a string and tests if it contains vowels
+   * @param {String} this
+   * @return {Boolean} true or false
+   */
   hasVowels() {
     return /[aeiou]/.test(this);
   },
 
-  toUpper() { },
+/**
+   * This method takes a string and returns it in uppercase
+   * @param {String} this
+   * @return {String} this
+   */
+  toUpper() {
+    return this.replace(/[a-z]/g, caps =>
+      String.fromCharCode(caps.charCodeAt(0) - 32)
+    );
+  },
 
-  toLower() { },
+/**
+   * This method takes a string and returns it in lowercase
+   * @param {String} this
+   * @return {String} this
+   */
+  toLower() {
+    return this.replace(/[A-Z]/g, lower =>
+      String.fromCharCode(lower.charCodeAt(0) + 32)
+    );
+  },
 
-  ucFirst() { },
+/**
+   * This method takes a string and returns its first letter in uppercase
+   * @param {String} this
+   * @return {String} this
+   */
+  ucFirst() {
+    return this.replace(this[0], this[0].toUpper());
+  },
 
-  isQuestion() { },
+/**
+   * This method takes a string and checks if it's a question
+   * @param {String} this
+   * @return {Boolean} true or false
+   */
+  isQuestion() {
+    return /\?$/.test(this);
+  },
 
-  words() { },
+/**
+   * This method takes a string and check if it has multiple words
+   * @param {String} this
+   * @return {Array} array
+   */
+  words() {
+    return this.replace(/[^\w\s]/g, '').split(/\s+/);
+  },
```

Comment:
- You may also want to test more edge cases here as well.  What will `"%$$33 ^^^ *** Nigeria need*s a mira%%%%cle!".words()` return?